1_fetch/src/my_happy_script.R

```diff
@@ -1,83 +1,108 @@
+## ---------------------------
```

Comment:
This commit creates four functions to easily run a workflow in R that  takes data from the web and generates a PNG, .txt. and .csv output in three distinct folders. 

Functions are:

getCSV(out_folder) - takes argument for path to save data to
eval_data(in_csv) - takes input data and creates a dataframe with required column names.
	
createPlot(evalData, plotOutput) - takes a dataframe and an
output file and generate a plot in that file.

dataRender(eval_dtaa, textOutput) - takes a dataframe and an output file and writes a descriptive piece of text about the
data contained within the dataframe.

to run the code, add each of the four functions to the
environment. Then run all the pieces of code that create the
desired output directories and then run the code that calls
these functions.